src/main/java/com/bynder/sdk/query/upload/SaveMediaQuery.java

```diff
@@ -87,12 +92,22 @@ public SaveMediaQuery setAudit(final Boolean audit) {
         return this;
     }
 
-    public String getMetaproperty() {
-        return metaproperty;
+    public SaveMediaQuery setMetaproperties(List<MetapropertyAttribute> metaproperties) {
+        this.metaproperties = metaproperties;
+        return this;
     }
 
-    public SaveMediaQuery setMetaproperty(String metaproperty) {
-        this.metaproperty = metaproperty;
+    public List<MetapropertyAttribute> getMetaproperties() {
+        return metaproperties;
+    }
+
+    public SaveMediaQuery setMetaproperty(String metapropertyUuid, String optionName) {
```

Comment:
`addMetaproperty` would describe a bit better what it does, now the logic has changed here.

But do we even intent to still use this method? Since we changed it to the plural form in FileUploader.java for example.

Reply:
Yes, one stakes a List of MetapropertyAttributes and one takes a key-value pair. The two address distinct use cases.

Reply:
In src/main/java/com/bynder/sdk/service/upload/FileUploader.java , you changed `setMetaproperty(uploadQuery.getMetaproperty())` into `setMetaproperties(uploadQuery.getMetaproperties())`. 

Apart from there, `SaveMediaQuery` is not used, and in a typical file upload use case you also wouldn't use it directly (but rather `UploadQuery` instead). So I don't really see the benefits of having this method, versus the potential confusion it could cause.

Reply:
done